scalikejdbc-mapper-generator-core/src/main/scala/scalikejdbc/mapper/CodeGenerator.scala

```diff
@@ -924,6 +981,12 @@ class CodeGenerator(table: Table, specifiedClassName: Option[String] = None)(imp
           |      val shouldBeNone = %className%.find(%primaryKeys%)
           |      shouldBeNone.isDefined should beFalse
           |    }
+          |    def batchInsert = this {
+          |      val list = %className%.findAll()
```

Comment:
@letusfly85 Sorry, I've misunderstood your code >%className%.findAll().foreach(%className%.destroy)

Please fix like this:

```scala
val entities = %className%.findAll()
entities.foreach(e => %className%.destroy(e))
val batchInserted = %className%.batchInsert(entities)
batchInserted.size should be_>(0)
```